src/Tribe/Views/V2/Views/Month_View.php

```diff
@@ -283,6 +286,7 @@ static function ( $event ) use ( $date_object ) {
 				'multiday_events'  => $day_stack,
 				'found_events'     => $day_found_events,
 				'more_events'      => $more_events,
+				'day_url'          => $day_url,
```

Comment:
I do like that approach. Wiill merge into mine

Reply:
Awesome! yeah it's what Luc4 suggested we want to start doing.